Dockerfile

```diff
@@ -11,7 +11,7 @@ LABEL org.opencontainers.image.source="https://github.com/renovatebot/renovate"
   org.opencontainers.image.licenses="AGPL-3.0-only"
 
 # renovate: datasource=node
-RUN install-tool node v14.19.1
+RUN install-tool node v16.14.2
```

Comment:
I think let's set ENV BINARY_SOURCE=install at the same time

Reply:
OK, we need to fix some buildpack stuff first